avm/res/app-configuration/configuration-store/tests/e2e/defaults/main.test.bicep

```diff
@@ -43,6 +45,9 @@ module testDeployment '../../../main.bicep' = [
     params: {
       name: '${namePrefix}${serviceShort}001'
       location: resourceLocation
+      enablePurgeProtection: true
+      disableLocalAuth: true
+      replicaLocations: ['eastus', 'westus']
```

Comment:
Defaults tests are supposed to only cover input parameters that are required. Ref https://azure.github.io/Azure-Verified-Modules/specs/bicep/#defaults-tests-must 

For this reason, I'd suggest to remove these optional parameters from this test, and add an exclusion to the defaults PSRule suppression group for the rule `Azure.AppConfig.GeoReplica`: https://github.com/Azure/bicep-registry-modules/blob/main/avm/utilities/pipelines/staticValidation/psrule/.ps-rule/min-suppress.Rule.yaml

Regarding `disableLocalAuth`, the PSRule Azure.AppConfig.DisableLocalAuth is not in the reliability pillar, thus non blocking for now. However, if we want to fix that already, I'd suggest to set its default value to `true` in the main.bicep template instead.

Reply:
For replicas, yes, absolutely. Ideally we should also add them to the max test, which usually covers all possible features and child resources to maximize parameter validation.

Reply:
Done! I also had to add one to the min rule on Audit Logs.